Source/Core/Core/PowerPC/JitArm64/JitArm64_BackPatch.cpp

```diff
@@ -62,7 +62,7 @@ void JitArm64::EmitBackpatchRoutine(u32 flags, bool fastmem, bool do_farcode, AR
     {
       if (flags & BackPatchInfo::FLAG_SIZE_F32)
       {
-        m_float_emit.FCVT(32, 64, D0, RS);
+        fpr.ConvertDoubleToSingleLower(D0, RS);
```

Comment:
Call here likely fails to save `gprs_to_push` where necessary before internal function call, ditto for all other instances

Reply:
Doesn't the call to `ABI_PushRegisters` inside the `Convert*` functions solve this, since `gprs_to_push` is always a (non-strict) subset of `gpr.GetCallerSavedUsed()`? Though, this did make me realize another problem: The slowmem path first does `ABI_PushRegisters` in this file and then `ABI_PushRegisters` again in `Convert*`, which is unnecessary. Perhaps it would make more sense to have a `gprs_to_push` argument for the `Convert*` functions than for the FPR cache to keep a reference to the GPR cache.

Reply:
Brain failure; thought this function was called during backpatching due to its name.